firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -527,7 +532,7 @@ private Void deleteFirebaseInstallationId() throws FirebaseInstallationsExceptio
             /*projectID= */ getProjectIdentifier(),
             /*refreshToken= */ entry.getRefreshToken());
 
-      } catch (FirebaseException exception) {
+      } catch (FirebaseException | IOException exception) {
```

Comment:
Why do we catch `FirebaseException` here?

a) Do we ever use any Exception type other than `FirebaseInstallationsException`?
If yes: why? If no: I recommend to check for `FirebaseInstallationsException` here.
b) Why do we catch `FirebaseInstallationsException`?
catching and rethrowing the same exception is an antipattern.
Can we not just leave the FirebaseInstallationsException as is?

Reply:
Update to check only `FirebaseInstallationsException`.

Agreed on the anti-pattern. Updated the code to remove this code.

I don't know the reason why we had this anti-pattern behavior before.